code/datums/elements/food/microwavable.dm

```diff
@@ -57,6 +57,12 @@
 	if(istype(result, default_typepath))
 		recipe_result |= COMPONENT_MICROWAVE_BAD_RECIPE
 
+	if(randomize_pixel_offset && isitem(result))
+		var/obj/item/result_item = result
+		if(!(result_item.item_flags & NO_PIXEL_RANDOM_DROP))
+			result_item.pixel_x = result_item.base_pixel_x + rand(-6, 6)
+			result_item.pixel_y = result_item.base_pixel_y + rand(-5, 6)
```

Comment:
why -5 and 6 and not -6 and 6

Reply:
Since microwaves only take up 2/3rds of a tile, -6 looked a little too far down when I was testing it.